src/components/common/animations/FaceVerificationSmiley/index.js

```diff
@@ -2,30 +2,21 @@ import React from 'react'
 import { Platform } from 'react-native'
 import Lottie from 'lottie-react-native'
 import AnimationBase from '../Base'
-import { getScreenHeight } from '../../../../lib/utils/Orientation'
-import { getAnimationData } from '../../../../lib/utils/lottie'
-
-const { animationData, imageAssetsFolder } = getAnimationData('FaceVerificationSmiley', require('./data'))
-
-const margin = getScreenHeight() > 700 ? 175 : getScreenHeight() / 2.3
+import animationData from './data.json'
 
 const styles = {
   android: {},
   ios: {},
-  web: {
-    marginTop: -margin,
-    marginBottom: -margin,
-  },
+  web: {},
 }
 
 class FaceVerificationSmiley extends AnimationBase {
   render() {
     return (
       <Lottie
-        imageAssetsFolder={imageAssetsFolder}
         enableMergePathsAndroidForKitKatAndAbove={true}
         autoPlay={true}
-        source={this.improveAnimationData(animationData)}
+        source={animationData}
```

Comment:
we dont need this?

Reply:
+1
@StanislavShevchenko There’s a bug in Lottie which brings to the memory leaks and, as the result to the browser’s hanging / crash

To avoid this internal bug we’re setting deep copy of the animation json

So, improveAnimationData is required

Reply:
@StanislavShevchenko told that improveAnimationData won't process data of this oner animation. I'll try to change its implementation to lodash's cloneDeep instead of json stringify then parse